README.md

```diff
@@ -1,43 +1,78 @@
-your own models and quickly experiment with different solutions.
+GluonTS is a Python package for probabilistic time series modeling, focusing on deep learning based models.
 
-* [Documentation (stable version)][stable docs url]
-* [Documentation (latest)][latest docs url]
-* [JMLR MLOSS Paper](http://www.jmlr.org/papers/v21/19-820.html)
-* [ArXiv Paper](https://arxiv.org/abs/1906.05264)
+## Features
 
-[stable docs url]: https://gluon-ts.mxnet.io/
-[latest docs url]: https://gluon-ts.s3-accelerate.dualstack.amazonaws.com/master/index.html
+* State-of-the-art models implemented with [MXNet](https://mxnet.incubator.apache.org/) and [PyTorch](https://pytorch.org/)
+* Utilities for loading and iterating over time series datasets
+* Utilities to evaluate models performance and compare their accuracy
+* Building blocks to define your own models and quickly experiment with different solutions
+* AWS integration via [Amazon SageMaker](https://aws.amazon.com/de/sagemaker/)
 
 ## Installation
 
 GluonTS requires Python 3.7, and the easiest way to install it is via `pip`:
 
 ```bash
-pip install --upgrade mxnet~=1.7 gluonts
+pip install --upgrade gluonts mxnet~=1.8   # to be able to use MXNet-based models
+pip install --upgrade gluonts torch~=1.10  # to be able to use PyTorch-based models
 ```
 
-## Dockerfiles
+## Documentation
 
-Dockerfiles compatible with Amazon Sagemaker can be found in the [examples/dockerfiles](https://github.com/awslabs/gluon-ts/tree/master/examples/dockerfiles) folder.
-
-## Quick start guide
+* [Documentation (stable version)][stable docs url]
+* [Documentation (development version)][development docs url]
+* [JMLR MLOSS Paper](http://www.jmlr.org/papers/v21/19-820.html)
+* [ArXiv Paper](https://arxiv.org/abs/1906.05264)
 
-This simple example illustrates how to train a model from GluonTS on some data,
-and then use it to make predictions. As a first step, we need to collect
-some data: in this example we will use the volume of tweets mentioning the
+## Implemented models
+
+Name                             | Local/global | Data layout              | Architecture | Implementation | References
+---------------------------------|--------------|-------------------|--------------|----------------|-----------
+DeepAR                           | Global       | Univariate               | RNN | [MXNet](src/gluonts/model/deepar/_estimator.py), [PyTorch](src/gluonts/torch/model/deepar/estimator.py) | [paper](https://doi.org/10.1016/j.ijforecast.2019.07.001)
+DeepFactor                       | Global       | Univariate               | RNN, state-space model, Gaussian process | [MXNet](src/gluonts/model/deep_factor/_estimator.py) | [paper](https://proceedings.mlr.press/v97/wang19k.html)
+DeepState                        | Global       | Univariate               | RNN, state-space model | [MXNet](src/gluonts/model/deepstate/_estimator.py) | [paper](https://papers.nips.cc/paper/2018/hash/5cf68969fb67aa6082363a6d4e6468e2-Abstract.html)
```

Comment:
`DeepState` should be above `DeepFactor`; makes sense that way both chronologically and architecture-wise.

Reply:
I fell back to alphabetical ordering when in doubt, but what you propose makes sense; maybe the link to the paper could mention the year ("Rangapuram et al., 2018") so that this is also apparent